backend/apps/cache/services/cache_manager.py

```
@@ -340,23 +340,73 @@ def wrapper(*args, **kwargs):
-    manager.invalidate_dependencies(instance)
+    if not _cache_signals_enabled:
+        return
+    # Skip during migrations or if contenttypes not ready
+    if _is_migration_in_progress():
+        return
+    try:
+        manager = CacheManager()
+        manager.invalidate_dependencies(instance)
+    except Exception:
+        # Silently ignore errors during migrations
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)